src/System.Net.HttpListener/src/System/Net/Windows/CookieExtensions.cs

```diff
@@ -15,11 +15,22 @@ public static string ToServerString(this Cookie cookie)
         {
             if (s_toServerStringFunc == null)
             {
-                s_toServerStringFunc = (Func<Cookie, string>)typeof(Cookie).GetMethod("ToServerString").CreateDelegate(typeof(Func<Cookie, string>));
+                s_toServerStringFunc = (Func<Cookie, string>)typeof(Cookie).GetMethod("ToServerString", BindingFlags.NonPublic | BindingFlags.Instance).CreateDelegate(typeof(Func<Cookie, string>));
             }
             return s_toServerStringFunc(cookie);
         }
 
+        private static Func<Cookie, Cookie> s_cloneFunc;
+
+        public static Cookie Clone(this Cookie cookie)
+        {
+            if (s_cloneFunc == null)
+            {
+                s_cloneFunc = (Func<Cookie, Cookie>)typeof(Cookie).GetMethod("Clone", BindingFlags.NonPublic | BindingFlags.Instance).CreateDelegate(typeof(Func<Cookie, Cookie>));
```

Comment:
Do we have an issue somewhere tracking exposing Clone (or equivalent functionality)?